src/core/Song.cpp

```diff
@@ -393,9 +388,7 @@ void Song::processNextBuffer()
 		framesPlayed += framesToPlay;
 		m_playPos[m_playMode].setCurrentFrame( framesToPlay +
 								currentFrame );
-		m_elapsedMilliSeconds += 
-			( ( framesToPlay / framesPerTick ) * 60 * 1000 / 48 ) 
-				/ getTempo();
+		m_elapsedMilliSeconds += MidiTime::ticksToMilliseconds( framesToPlay / framesPerTick, getTempo());
```

Comment:
I think this line caused bug. Possible fix is: change
`double MidiTime::ticksToMilliseconds(tick_t ticks, bpm_t beatsPerMinute)` to
`double MidiTime::ticksToMilliseconds(double ticks, bpm_t beatsPerMinute)`.